main.test.js

```diff
@@ -0,0 +1,38 @@
+const readWorld = require('./main');
+const calculateWordsStatistic = require('./main');
+
+test('string in the number of words', () => {
+	expect(calculateWordsStatistic(readWorld('testData')).toEqual
```

Comment:
No need to import `./main` here, just pass test string to this function like:

```
test('check simple text statistic', () => {
  const testText = "Some test text, lala, test";
  expect(calculateWordsStatistic(testText)).toEqual({'some': 1, 'test': 1, 'text': 2, 'lala': 1})
}
```

Reply:
Changes are made